raiden/tests/utils/network.py

```diff
@@ -30,6 +31,19 @@
 )
 
 
+def dict_merge(to_update, other_dict):
```

Comment:
Does it work? Because that is trying to modify the dictionary while iterating over it:

https://github.com/raiden-network/raiden/blob/ae3c18297b0f166e8ceb016ef48550090a8a8b8e/raiden/utils/__init__.py#L308

Reply:
It isn't iterating over `a`, but over `b`, so yes, it works, but that `path` logic remained from an old logic, it doesn't affect the behavior, so can be removed

Reply:
> It isn't iterating over a, but over b

https://github.com/raiden-network/raiden/blob/ae3c18297b0f166e8ceb016ef48550090a8a8b8e/raiden/utils/__init__.py#L302

Reply:
> It isn't iterating over a, but over b

True, I misread `if key in a:` as `for key in a:`